x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/__snapshots__/trusted_apps_page.test.tsx.snap

```diff
@@ -1349,7 +1349,7 @@ Object {
                                   class="euiButtonEmpty__text"
                                 >
                                   Rows per page
-                                  : 
+                                  :
```

Comment:
Were changes to this file made intentionally?

Reply:
This file has been removed as when I merged in master so it's taken care of.